src/BlockTypes/MiniCart.php

```diff
@@ -332,4 +367,22 @@ protected function get_cart_item_markup() {
 			</td>
 		</tr>';
 	}
+
+	/**
+	 * Get the supports array for this block type.
+	 *
+	 * @see $this->register_block_type()
+	 * @return string;
+	 */
+	protected function get_block_type_supports() {
```

Comment:
I'm not sure to understand why this is needed either.

Reply:
This ensures the global style generated on the front end match with the global style in the Site Editor. We need this because of the `__experimentalSelectors`.